scenes/game/local_game.gd

```diff
@@ -933,6 +935,36 @@ class Player:
 			deck.insert(0, card)
 		return events
 
+	func get_unknown_cards():
+		var unknown_cards = hand + deck # TODO: secret sealed areas
+		parent.shuffle_array(unknown_cards)
```

Comment:
This is going to screw things up, you're using the random seed only on one client since you're calling this from game_wrapper but only for one player, right?

Reply:
oh good point; i was thinking of just sorting by ID instead so that's probably a good reason to do so